libcxx/include/__filesystem/path.h

```diff
@@ -14,9 +14,8 @@
 #include <__algorithm/replace_copy.h>
```

Comment:
Not attached to this file: You should update `libcxx/utils/generate_iwyu_mapping.py`, which currently contains

```
elif header == "__fwd/hash.h":
        return ["functional"]
```

Generally speaking, make sure you grep for `__fwd/hash.h` to make sure you find any stray references.